File: service-center-app/src/app/models/ServiceCategory.ts
export class ServiceCategory {

    category_ID: number;
    category_NAME: string;

    constructor(json?: any) {
        if (json) {
            this.category_ID = json.category_ID;
            this.category_NAME = json.category_NAME;
        }
        else{
            this.category_ID = 0;
            this.category_NAME = "";
        }
    }
}

File: src/main/java/com/dbms/team15/models/ServiceCategory.java
package com.dbms.team15.models;

public class ServiceCategory {

  private int CATEGORY_ID;
  private String CATEGORY_NAME;

  public ServiceCategory() {}

  public ServiceCategory(int CATEGORY_ID, String CATEGORY_NAME) {
    this.CATEGORY_ID = CATEGORY_ID;
    this.CATEGORY_NAME = CATEGORY_NAME;
  }

  public int getCATEGORY_ID() {
    return CATEGORY_ID;
  }

  public String getCATEGORY_NAME() {
    return CATEGORY_NAME;
  }

  public void setCATEGORY_ID(int CATEGORY_ID) {
    this.CATEGORY_ID = CATEGORY_ID;
  }

  public void setCATEGORY_NAME(String CATEGORY_NAME) {
    this.CATEGORY_NAME = CATEGORY_NAME;
  }

  @Override
  public String toString() {
    return (
      "ServiceCategory{" +
      "CATEGORY_ID=" +
      CATEGORY_ID +
      ", CATEGORY_NAME='" +
      CATEGORY_NAME +
      '\'' +
      '}'
    );
  }
}
